runtime/gc_ctrl.c

```diff
@@ -129,7 +129,7 @@ CAMLprim value caml_gc_get(value v)
   res = caml_alloc_tuple (11);
   Store_field (res, 0, Val_long (Caml_state->minor_heap_wsz));  /* s */
   Store_field (res, 2, Val_long (caml_percent_free));           /* o */
-  Store_field (res, 3, Val_long (caml_params->verb_gc));        /* v */
+  Store_field (res, 3, Val_long (caml_verb_gc));        /* v */
```

Comment:
`Val_long (caml_verb_gc)` does a non-atomic read. 

```suggestion
  Store_field (res, 3, Val_long (atomic_load_relaxed(&caml_verb_gc)));        /* v */
```

The indentation of the comment needs to be fixed.